packages/interactor/src/app.ts

```diff
@@ -8,14 +8,28 @@ export const App = {
       if(!appUrl) {
         throw new Error('no app url defined');
       }
-      let testFrame = bigtestGlobals.testFrame;
+      // eslint-disable-next-line prefer-let/prefer-let
+      const testFrame = bigtestGlobals.testFrame;
       if(!testFrame) {
         throw new Error('no test frame defined');
+      } else {
+        let url = new URL(appUrl);
+        url.pathname = path;
+        testFrame.src = url.toString();
+        await new Promise((resolve, reject) => {
```

Comment:
I tried that, but it didn't seem to work, unfortunately. I think there is a race condition where the readyState is `complete` before actually flipping to `loading`.

Also, the polling approach of `converge` will be much more processor intensive during a time when the application will want any resources it can get, so this could also have a performance impact.

It's really telling here how useful effection is, the cleanup needed for just these TWO simulataneous async processes so they don't get messed up is pretty terrible.